README.md

```diff
@@ -1,37 +1,62 @@
 # Exercism Elm Track
-[![build status](https://github.com/exercism/elm/workflows/elm%20%2F%20master/badge.svg)]
```

Comment:
There was an outer pair of square brackets showing on the github ui, so i removed them